packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -170,6 +174,13 @@ const Breakouts = WebexPlugin.extend({
    */
   updateBreakout(params) {
     this.set(params);
+    if (params.sessionType === BREAKOUTS.SESSION_TYPES.MAIN) {
```

Comment:
issue: It is not necessary to store the main session ID or main group ID separately. Either the user is in the main session, in which case the sessionId, groupId are on this object or they are in a session, in which case you can filter the breakouts by isMain to determine which groupId, sessionId to use.

A function similar to this could be used to retrieve the information:

```ts
  getMainSession() {
    if (this.isInMainSession) {
      return this;
    }

    const mainSession = this.parent.breakouts.filter((breakout) => breakout.isMain)[0];

    return mainSession;
  },
```

There is similar logic used in the leave function in breakout,ts so this function could possibly be shared.

Reply:
Done. Please help to check the PR again